bandersnatch/fp/element.go

```diff
@@ -761,7 +761,6 @@ func (z *Element) SetBigInt(v *big.Int) *Element {
 	vv := bigIntPool.Get().(*big.Int)
 
 	// copy input + modular reduction
-	vv.Set(v)
```

Comment:
I noticed this assignment shouldn't be needed since in the next line it will be used to store the `Mod(..)` result. (Unless I'm missing something here).

We're in generated code ground, so we still have this tension of what to do around here, but this is in many hot paths. 
I'm open to rever this change to keep avoiding touching this, but from another angle is quite unfortunate to also keep this unnecessary work being done here.